api/samples/memval_simple.c

```diff
@@ -403,7 +421,7 @@ event_exit(void)
 DR_EXPORT void
 dr_client_main(client_id_t id, int argc, const char *argv[])
 {
-    drreg_options_t ops = {sizeof(ops), 3 /*max slots needed*/, false};
+    drreg_options_t ops = {sizeof(ops), 3 /*max slots needed*/, true};
```

Comment:
Wait, I missed this part of the change -- why do we need this?  I thought memval_simple managed to work within drreg's framework by not inserting post-instr instru until it hit the next instr, so it was always doing pre-instr instru, which is what drreg assumes.